hub/api/dataset.py

```diff
@@ -143,10 +143,8 @@ def create_tensor(
         self,
         name: str,
         htype: str = DEFAULT_HTYPE,
-        chunk_size: int = None,
-        dtype: Union[str, np.dtype, type] = None,
-        sample_compression: str = None,
-        chunk_compression: str = None,
+        dtype: Union[str, np.dtype, type] = UNSPECIFIED,
+        sample_compression: str = UNSPECIFIED,
```

Comment:
This is the new behaviour right?
If a user passes `sample_compression = None`, it's always uncompressed but if they don't specify compression, it might use the default compression of the htype.

Reply:
that's a different PR

Reply:
wait it should have been updated here

Reply:
only when that tensor is of `htype="image"`

Reply:
#1038